src/Stryker.CLI/Stryker.CLI.UnitTest/Stryker.CLI.UnitTest.csproj

```diff
@@ -22,8 +22,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="stryker-config.json" CopyToOutputDirectory="Always" />
-    <None Include="filled-stryker-config.json" CopyToOutputDirectory="Always" />
+    <None Update="filled-stryker-config.json" CopyToOutputDirectory="Always" />
```

Comment:
The [docs ](https://docs.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild?view=vs-2017) say this about the Update attribute:

> Enables you to modify metadata of a file that was included by using a glob.

This does not seem to do what we want to happen to this file. What is the purpose of this change?